apps/bfd-pipeline/bfd-pipeline-rda-bridge/src/main/java/gov/cms/bfd/pipeline/bridge/io/NdJsonSink.java

```diff
@@ -28,5 +38,19 @@ public void write(MessageOrBuilder messageOrBuilder) {
   @Override
   public void close() throws IOException {
     writer.close();
+
```

Comment:
Could WrappedCounter just track its min/max values so you don't need this deep knowledge of the message objects?